docs/rules/saga-direct-delegation.md

```diff
@@ -0,0 +1,58 @@
+# Enforce using `yield*` to delegate to other sagas
+# saga-direct-delegation
+
+This rule enforces that delegation to sagas is done directly using `yield*`, rather than indirectly by plain `yield` of the returned iterator to the saga middleware.

```

Comment:
The premise of this rule is that `yield*`ing to a delegate iterator is equivalent to `yield`ing that same iterator to the middleware to be iterated. This would allow you to safely let this rule autofix your entire existing codebase.

- [ ] Extend this paragraph to make this clear, so people aren't in doubt

Reply:
867c1fd